eisbuk-admin/src/components/atoms/CustomerList/CustomerDialog.tsx

```diff
@@ -0,0 +1,221 @@
+import { useTranslation } from "react-i18next";
+import { useDispatch } from "react-redux";
+
+import IconButton from "@material-ui/core/IconButton";
+import Box from "@material-ui/core/Box";
+import Typography from "@material-ui/core/Typography";
+import Card from "@material-ui/core/Card";
+import CardActions from "@material-ui/core/CardActions";
+import CardContent from "@material-ui/core/CardContent";
+import Dialog from "@material-ui/core/Dialog";
+
+import makeStyles from "@material-ui/core/styles/makeStyles";
+
+import DeleteIcon from "@material-ui/icons/Delete";
+import EditIcon from "@material-ui/icons/Edit";
+import DateRangeIcon from "@material-ui/icons/DateRange";
+import Mail from "@material-ui/icons/Mail";
+
+import { Customer } from "eisbuk-shared";
+
+import { Routes } from "@/enums/routes";
+import { CategoryLabel, CustomerLabel, Prompt } from "@/enums/translations";
+
+import EisbukAvatar from "@/components/users/EisbukAvatar";
+import ConfirmDialog from "@/components/global/ConfirmDialog";
+import CustomerForm from "@/components/customers/CustomerForm";
+
+import {
+  deleteCustomer,
+  sendBookingsLink,
+  updateCustomer,
+} from "@/store/actions/customerOperations";
+
+import { capitalizeFirst } from "@/utils/helpers";
+import {
+  __customerDeleteId__,
+  __customerEditId__,
+  __openBookingsId__,
+  __sendBookingsEmailId__,
+} from "./__testData__/testIds";
+
+interface Props {
+  customer: Customer;
+  onClick?: (customer: Customer) => void;
+  onClose: () => void;
+  open: boolean;
+}
+
+const CustomerDialog: React.FC<Props> = ({ open, onClose, customer }) => {
+  const { t } = useTranslation();
+  const classes = useStyles();
+
+  return (
+    <Dialog open={open} onClose={onClose}>
+      <Card className={classes.Card}>
+        <CardContent className={classes.CardContent}>
+          <Box className={classes.Box}>
+            {/* {extended && <AdditionalButtons {...customer} />} */}
+            <EisbukAvatar
+              {...{ ...customer, className: classes.BiggerAvatar }}
+            />
+            <Typography variant="h4" className={classes.Name}>
+              {customer.name} {customer.surname}
+            </Typography>
+          </Box>
+
+          <br></br>
+          <Typography variant="h6" component="div">
+            {t(CustomerLabel.Category)}:{" "}
+            {capitalizeFirst(t(CategoryLabel[customer.category])).replace(
+              "-",
+              " "
+            )}
+          </Typography>
+          <Typography variant="h6">
+            {t(CustomerLabel.Email)}: {customer.email}
+          </Typography>
+        </CardContent>
+        <CardActions className={classes.CardActions}>
+          <AdditionalButtons customer={customer} onClose={onClose} />
+        </CardActions>
+      </Card>
+    </Dialog>
+  );
+};
+
+/**
+ * Action buttons for customer operations, used only on `extended` variant.
+ */
+const AdditionalButtons: React.FC<Omit<Props, "open">> = ({
```

Comment:
This can be renamed to `ActionButtons`, and the function comment corrected (and you should always check that when reusing some code)